TestProject/output/GIFOutput.php

```diff
@@ -69,4 +75,47 @@ function finishOutput()
         }
     }
 
+    /**
+     * Defines output-specific parameters, which are required when calling one of the following options ->
+     * \"gifCellSize\ or \"gifCellColor\"
+     *
+     * @param Getopt $_options Defines output-specific parameters
+     */
+    function startOutput(Getopt $_options)
+    {
+        if ($_options->getOption("gifCellSize"))
+        {
+            $this->cellSize = intval($_options->getOption("gifCellSize"));
+
+            if ($this->cellSize <= 0)
+                $this->cellSize = 5;
+        }
+
+        if ($_options->getOption("gifCellColor"))
+        {
+            $this->gifCellColor = explode(",", $_options->getOption("gifCellColor"));
+        }
+    }
+
+
+    /**
+     *\"gifCellSize\" set the size for each cell and needs to be defined in pixel format -> 10 for example.
+     * Short option command: -p 10
+     * Long option command: --gifCellSize
+     *
+     * \"gifCellColor\" set the color for each cell where the colors for red, green, blue needs a number -> between 0-255
+     * and seperated with a comma.
+     * Short command: -c 155,155,155
+     * Long command: -gifCellColor 155,155,155
+     *
+     * @param Getopt $_options Defines output-specific options
+     */
+    function addOptions(Getopt $_options)
+    {
+        $_options->addOptions(
+            [
+                ['p', "gifCellSize", Getopt::REQUIRED_ARGUMENT, " Set the size of a single cell inside of a gif-output in pixel format"],
+                ['c', "gifCellColor", Getopt::REQUIRED_ARGUMENT, " Set the color fo a single cell inside a gif-output"]
```

Comment:
As discussed, please explain how the format of the color is. rgb-comma seperated?, hex? etc.
You explained it in the method which is very good, but also the user of the tool must know it.